device/marvell/arm64-marvell_db98cx8580_32cd-r0/plugins/sfputil.py

```diff
@@ -31,9 +26,10 @@ class SfpUtil(SfpUtilBase):
     _qsfp_ports = list(range(_port_start, ports_in_block + 1))
 
     def __init__(self):
-        os.system("modprobe i2c-dev")
+        subprocess.run(["modprobe", "i2c-dev"])
         if not os.path.exists("/sys/bus/i2c/devices/0-0050"):
-            os.system("echo optoe2 0x50 > /sys/bus/i2c/devices/i2c-0/new_device")
+            with open("/sys/bus/i2c/devices/i2c-0/new_device", 'w') as f:
+                subprocess.run(["echo", "optoe2", "0x50"], stdout=f)
```

Comment:
>"echo", "optoe2", "0x50" [](http://example.com/codeflow?start=32&length=24)

I see the original intention is to write a string to a file, so there is no need to invoke subprocess at all.

Reply:
All fixed, please check again.

Reply:
Done.